src/modules/emote_menu/components/Button.jsx

```diff
@@ -0,0 +1,27 @@
+import React, {useRef} from 'react';
+import {faSmile} from '@fortawesome/free-solid-svg-icons/faSmile';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Whisper from 'rsuite/lib/Whisper/index.js';
+import Popover from 'rsuite/lib/Popover/index.js';
+import EmoteMenu from './Menu.jsx';
+import styles from '../styles/button.module.css';
+
+export default function Button() {
+  const triggerRef = useRef(null);
+
+  return (
+    <Whisper
+      trigger="click"
+      placement="topEnd"
+      speaker={
+        <Popover className={styles.popover} full>
+          <EmoteMenu triggerRef={triggerRef} />
+        </Popover>
+      }
+      triggerRef={triggerRef}>
+      <button type="button" className={styles.button}>
+        <FontAwesomeIcon icon={faSmile} />
```

Comment:
this icon doesn't appear as the same size as twitch's native one. im also curious if there's a way to make use of twitch's?